### PR TITLE
Resolve Property name to color

### PR DESCRIPTION
**Pull request by ggetz**
Resolves a property name to a color.

- [x] Add Unit Test

### CODE REVIEW

**Comment by ggetz** [comment]
Updated and added specs.

**Comment by pjcozzi** [comment]
Merge in `3d-tiles-style`.  There is a conflict (minor, I imagine).

File: Specs/Scene/ColorExpressionSpec.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    it('marks style engine dirty', function() {
+        spyOn(MockStyleEngine.prototype, 'makeDirty');
+
+        var expression = new ColorExpression(new MockStyleEngine(), 'colorProperty');
+        expression.propertyName = 'newColorProperty';
+
+        expect(MockStyleEngine.prototype.makeDirty).toHaveBeenCalled();
+    });
+
+    it('does not marks style engine dirty', function() {
```
Comment (pjcozzi) [inline Specs/Scene/ColorExpressionSpec.js]: Throughout, this should be "does not mark style..."

**Comment by pjcozzi** [comment]
Looks good!

**Comment by ggetz** [comment]
Updated

**Comment by pjcozzi** [comment]
:+1:

**Pull request merged**